clients/drcachesim/tracer/raw2trace.h

```diff
@@ -1268,8 +1274,10 @@ template <typename T> class trace_converter_t {
                 buf->type = instr->prefetch_type();
                 buf->size = 1;
             } else if (instr->is_flush()) {
-                buf->type = TRACE_TYPE_DATA_FLUSH;
-                buf->size = (ushort)opnd_size_in_bytes(opnd_get_size(memref.opnd));
+                buf->type = instr->flush_type();
+                // i#4398: Handle flush sizes larger than ushort.
```

Comment:
style: as is this is just a descriptive comment of the code.  It needs a `TODO` or at least `XXX` so we know this is referring to unfinished work.  Ditto elsewhere.

Reply:
Done.